src/dotnet/commands/dotnet-help/xlf/LocalizableStrings.pt-BR.xlf

```diff
@@ -199,12 +199,12 @@
       </trans-unit>
       <trans-unit id="SDKListSdksCommandDefinition">
         <source>Display the installed SDKs.</source>
-        <target state="new">Display the installed SDKs.</target>
+        <target state="translated">Exiba os SDKs instalados.</target>
         <note />
       </trans-unit>
       <trans-unit id="SDKListRuntimesCommandDefinition">
         <source>Display the installed runtimes.</source>
-        <target state="new">Display the installed runtimes.</target>
+        <target state="translated">Exiba os tempos de execução instalados.</target>
```

Comment:
This translation is wrong. The runtime here is not the run time. It is a single word and I am not sure it translates.

Reply:
@janbaltus @v-zbsail Should this translation employ the "Runtimes" loan word similar to the German translation?

Reply:
The term “runtime” or “run time” can be used in two contexts: 

- The engine at the core of managed code execution. The runtime supplies managed code with services such as cross-language integration, code access security, object lifetime management, and debugging and profiling support. (TS)

- time during which a program is running (executing), in contrast to other program lifecycle phases such as compile time, link time and load time. (wikipedia)


Translation: Exiba os tempos de execução instalados.

NEW Translation: Exiba os runtimes instalados.

We are updating this PR with fix

Reply:
PR updated with the fix